app/models/verification_document.rb

```diff
@@ -31,7 +31,15 @@ def add_header(pdf)
 
   def header
     output = "Name:                  #{benefit_application.display_name}\n"
-    output += "Date of Birth:        #{benefit_application.birthday}\n"
+    output += "Date of Birth:        #{birthday}\n"
     output
   end
+
+  def birthday
+    if benefit_application.primary_member.birthday.present?
+      benefit_application.primary_member.formatted_birthday
```

Comment:
I'm uncomfortable with the formatted_birthday as an attribute on model, but don't think this should block this PR! would like to talk about how our team wants to handle model vs view concerns going forward

Reply:
yeah agreed! we have to deal with a `nil` case on birthday somewhere else in our PDF parsing code as well and it's icky